pg_search/README.md

```diff
@@ -204,12 +207,27 @@ cargo pgrx init --pg16=/usr/local/opt/postgresql@16/bin/pg_config
 
 # Ubuntu
 cargo pgrx init --pg16=/usr/lib/postgresql/16/bin/pg_config
+
+# Arch
+cargo pgrx init --pg16=/usr/bin/pg_config
 ```
 
 If you prefer to use a different version of Postgres, update the `--pg` flag accordingly.
 
 Note: While it is possible to develop using pgrx's own Postgres installation(s), via `cargo pgrx init` without specifying a `pg_config` path, we recommend using your system package manager's Postgres as we've observed inconsistent behaviours when using pgrx's.
 
+`pgrx` requires `libclang`. To install it:
+
+```bash
```

Comment:
Can you add macos here too, and do only libclang-dev?

Reply:
> do only libclang-dev

Done.

> Can you add macos here too

As mentioned in a previous comment I don't have a Mac to test against at the moment.

https://github.com/pgcentralfoundation/pgrx/?tab=readme-ov-file#system-requirements doesn't mention installing libclang on macOS which leads me to wonder if it's preinstalled.

Searching Stack Overflow it looks like Clang is installed with Xcode.

What would you like me to put? Perhaps: "For macOS please install Xcode."

Reply:
You're right, let's just omit macOS clang comes by default. My bad